cf/app/help.go

```diff
@@ -105,6 +105,7 @@ func newAppPresenter(app *cli.App) (presenter appPresenter) {
 					presentCommand("logout"),
 					presentCommand("passwd"),
 					presentCommand("target"),
+					presentCommand("user-info"),
```

Comment:
I think 'user-info' command can not go with 'GETTING STARTED' group of commands. Please check.